### DISCUSSION TITLE
IOS no permissions to update labels

### DISCUSSION THREAD

**Discussion by LeeLenaleee** [community/community#13654] [Mobile]
When using the iOS app and you try to update labels of an issue in a repo that you have triage access to it will give an error that you don't have the correct permissions. When I try it with the same account using the website version it works fine

**Comment by Wolvo666** [comment]
#13303

**Reply by LeeLenaleee** [reply]
This has nothing to do with my issue, the issue you linked is about co-pilot while my issue is with the iOS app of github

**Comment by Jay-o-Way** [comment]
I have the **exact** same problems, just in Android.
